warehouse/templates/pages/help.html

```diff
@@ -178,6 +178,9 @@ <h3 id="compromised-password">{{ compromised_password() }}</h3>
           <p>
             If you receive an error message saying that "This password has appeared in a breach or has otherwise been compromised and cannot be used", you should change it all other places that you use it as soon as possible.
```

Comment:
Do we need to update the wording of the quoted error message here?

Reply:
Yup, good catch.